security/pfSense-pkg-suricata/files/arm/usr/local/www/suricata/suricata_rulesets.php

```diff
@@ -428,9 +428,9 @@
 						</td>
 						<td colspan="4">
 						<?php if ($no_community_files): ?>
-							<?php echo gettext("{$msg_community}"); ?>
+							<?php echo gettext($msg_community); ?>
```

Comment:
`$msg_community` declaration already passed through `gettext()` at https://github.com/pfsense/FreeBSD-ports/blob/0d68a01c5e7180e9afe9b06265391aa84103fc58/security/pfSense-pkg-suricata/files/arm/usr/local/www/suricata/suricata_rulesets.php#L407 so it could be removed on echo commands

Reply:
Changes made.  Sorry I missed that earlier.